Libraries/IntegrationServiceLibrary.psm1

```diff
@@ -298,12 +298,12 @@ function Wait-ForVMToStartKVP {
 	param (
 		[String] $VmName,
 		[String] $HvServer,
-		[int] $StepTimeout
+		[int] $StepTimeout = 30
 	)
 	$ipv4 = $null
 	$retVal = $False
 
-	$waitTimeOut = $StepTimeout
+	$waitTimeOut = ($StepTimeout,50 | Measure -Min).Minimum
```

Comment:
What makes you recommend 25-min timeout? It seems to be long to communicate to the host.

Reply:
In fact, 5 mins are enough in my experience, but I'm not sure there is an exception.

Reply:
5 min for IP address obtain is also long. But as 'time-out' value, I am ok with it. Please fix it with 5-min. @chvalean @LiliDeng please comment for 5-min time-out change.

Reply:
@xian123 please do not mark as 'resolved' unless you are clearly communicating out or gaining the information. I bring up 5-min timeout for IP address obtain, there seems to be no discussion. You should get the answer before marking it.

Reply:
@xian123 changed? @LiliDeng consult with @xian123 for close.

Reply:
We are testing it in upstream pipeline.